[DOCS-8802] adding Mobile apps to restricted Networks (#25265)
DataDog/documentation: content/en/synthetics/mobile_app_testing/mobile_app_tests/restricted_networks.md

---
title: Run Mobile App tests from Restricted Networks
description: "Run Mobile App tests from restricted networks"
further_reading:
- link: "https://www.datadoghq.com/blog/test-creation-best-practices/"
  tag: "Blog"
  text: "Best practices for creating end-to-end tests"
- link: "/synthetics/mobile_app_testing/mobile_app_tests"
  tag: "Documentation"
  text: "Learn how to create Synthetic mobile app tests"
- link: "/synthetics/mobile_app_testing/settings"
  tag: "Documentation"
  text: "Learn how to upload your iOS or Android mobile applications"
cascade:
  algolia:
    tags: ['mobile_testing']
---

## Overview

> Some of your applications may not be available to the public internet because they are either accessing development or local environments, or they are internal applications intended for users within your corporate network, for example, your corporate intranet or VPN. 

Reviewer: ```suggestion
Some of your applications might not be available to the public Internet because they are accessing development or local environments, or they are internal applications intended for users within your corporate network (for example, your corporate intranet or VPN). 
```

{{< img src="/mobile_app_testing/mobile_app_restricted_networks.png" alt="Diagram showing testing of mobile apps behind a firewall or restricted networks" style="width:100%;">}}

> To test these applications, add the following IP address ranges to your company's allow list to ensure successful requests from your applications in Datadog Mobile App Testing. 

Reviewer: ```suggestion
To test these applications, add the following IP address ranges to your company's allowlist. This ensures successful requests from your applications in Datadog Mobile App Testing. 
```
